src/improvements/tasks/MultisigTask.sol

```diff
@@ -656,15 +657,69 @@ abstract contract MultisigTask is Test, Script {
         }
     }
 
-    /// @notice print the tenderly simulation link with the state overrides
-    function printTenderlySimulationLink(Action[] memory actions) internal view {
-        Simulation.StateOverride[] memory overrides = new Simulation.StateOverride[](1);
-        overrides[0] =
+    /// @notice print the tenderly simulation payload with the state overrides
+    function printTenderlySimulationData(Action[] memory actions) internal view {
+        Simulation.StateOverride[] memory stateOverrides = new Simulation.StateOverride[](1);
+        stateOverrides[0] =
             Simulation.overrideSafeThresholdOwnerAndNonce(parentMultisig, msg.sender, _getNonce(parentMultisig));
         bytes memory txData = _execTransationCalldata(
             parentMultisig, getMulticall3Calldata(actions), Signatures.genPrevalidatedSignature(msg.sender)
         );
-        Simulation.logSimulationLink({_to: parentMultisig, _data: txData, _from: msg.sender, _overrides: overrides});
+
+        // Log the Tenderly JSON payload
+        console.log("\nSimulation payload:");
+        logTenderlySimulationPayload(txData, stateOverrides[0].overrides);
+
+        // Log the simulation link
+        console.log("\nSimulation link:");
+        Simulation.logSimulationLink({_to: parentMultisig, _data: txData, _from: msg.sender, _overrides: stateOverrides});
+
+        // Calculate domain separator
+        // TODO: Read from AddressRegistry for some addresses that have non compliant domain separators, such as the mainnet Foundation Operations Safe
```

Comment:
Alternative option here is just to pass in the Safe version. Versions < 1.3.0 have a different domain separator, everything else uses the newer separator that includes chain ID.

Reply:
That was a bit of a side quest, but I came up with a minimal implementation to check whether the Safe version is below 1.3.0